AirCasting/Onboarding/PrivacyOnboarding.swift

```diff
@@ -0,0 +1,115 @@
+// Created by Lunar on 09/06/2021.
+//
+
+import SwiftUI
+
+struct PrivacyOnboarding: View {
+    var completion: () -> Void
+    @State var presentingModal = false
+    var body: some View {
+        VStack(alignment: .leading, spacing: 40) {
+            progressBar
+            Spacer()
+            titleText
+            descriptionText
+            VStack(spacing: 5) {
+                continueButton
+                learnMoreButton
+            }
+        }
+        .padding()
+        .navigationBarHidden(true)
+    }
+    
+    struct ModalView: View {
+        @Binding var presentedAsModal: Bool
+        var body: some View {
+            VStack(alignment: .leading, spacing: 20) {
+                HStack(alignment: .top) {
+                    Spacer()
+                    Button(action: { presentedAsModal = false }) {
+                        HStack {
+                            Image(systemName: "xmark")
+                        }
+                        .padding()
+                    }
+                }
+                sheetTitle
+                sheetDescription
+                Spacer()
+            }
+        }
+
+        private var sheetTitle: some View {
```

Comment:
horizontal padding is missing :) 

<img width="298" alt="Screenshot 2021-06-28 at 11 23 14" src="https://user-images.githubusercontent.com/46443841/123612906-42abf800-d803-11eb-855a-926a94b127af.png">